code/io/keycontrol.cpp

```diff
@@ -218,6 +218,7 @@ static struct Cheat cheatsTable[] = {
 
 int Tool_enabled = 0;
 bool Perspective_locked=false;
+bool Slew_locked=false;
```

Comment:
Why is this in keycontrol and not playercontrol?

Reply:
Simply because `Perspective_locked` is also here.  If there's a better place to put them, then I'd like to move both of them together.